cogs/logs.py

```diff
@@ -0,0 +1,106 @@
+import nextcord
+from nextcord.ext import commands
+import aiosqlite
+import asyncio
+from datetime import datetime
+
+class Logs(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print("Online!")
+        setattr(self.client, "db", await aiosqlite.connect("main.db"))
+        await asyncio.sleep(1)
+        async with self.client.db.cursor() as cursor:
+            await cursor.execute("CREATE TABLE IF NOT EXISTS logging (channel INTEGER, guild INTEGER)")
+        await self.client.db.commit()
+
+    @commands.Cog.listener()
+    async def on_message_delete(self, message):
```

Comment:
iirc, this is dependent on the message being cached, if its not this would probably raise an error since `message.author` does not exist.